pkgs/by-name/sc/scipopt-ug/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  cmake,
+  scipopt-scip,
+  zlib,
+  mpi,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scipopt-ug";
+  version = "1.0.0-beta6";
+
+  # Take the SCIPOptSuite source since no other source exists publicly.
+  src = fetchzip {
+    url = "https://github.com/scipopt/scip/releases/download/v920/scipoptsuite-9.2.0.tgz";
+    sha256 = "SaIo/U9cconmGRxEPV5M7IO6+L9wGCaXqxifefUzFk4=";
+  };
+
+  sourceRoot = "${src.name}/ug";
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    scipopt-scip
+    mpi
+    zlib
+  ];
+
+  meta = {
+    maintainers = with lib.maintainers; [ fettgoenner ];
```

```suggestion
    maintainers = with lib.maintainers; [ fettgoenner ];
    changelog = "https://scipopt.org/doc-${scipVersion}/html/RN${lib.versions.major scipVersion}.php";
```
Same for other packages.

---

Will add this for each package.